docs/peopleops/offboarding/offboarding-process.md

```diff
@@ -0,0 +1,46 @@
+# Offboarding process
+
+Handover and offboarding of employees is an important process for a number of reasons:
+
+- data security and risk management;
+- efficient organisation and company administration;
+- a chance to get final feedback and review from exiting members; and
+- a opportunity to put the company in a strong position to continue developing projects that run
+  across capstone trimesters.
+
+## Update and correctly store all relevant documents
+
+Ensure that all projects' documentation and processes are up-to-date and have been stored in the
+correct location of the relevant [Thoth Tech repository](https://github.com/thoth-tech) on GitHub.
+Documentation should be clear enough for subsequent members to easily follow, understand and, as
+necessary, continue work on projects.
+
+## Handover Document
+
+Complete handover document and showcase according to the capstone requirements for the relevant
+trimester. Ensure ALL relevant PR or document links are included. Ensure a markdown copy of the
+handover document is uploaded to the relevant
+[Thoth Tech repository](<(https://github.com/thoth-tech)>)
+
+## Exit Survey
+
```

Comment:
should add a comment stating when these were updated last.
Same for onboarding forms

Reply:
Would the Git history and timestamp be sufficient? 🤔

Reply:
Added.